### PR TITLE
🐛 hack/release-notes: ensure relase notes tool can be used for external projects again

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:
Ensures the release note tool can again be used by other projects (in my case CAPV)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by sbueringer** [comment]
/hold

want to verify if it works before merge

cc @killianmuldoon

**Review by chrischdi — APPROVED** [review]
/lgtm

**Review by fabriziopandini — COMMENTED** [review]
a small nit, otherwise lgtm

**Comment by killianmuldoon** [comment]
/lgtm
/approve

/hold

Until you're happy with it.

@nawazkh we should add a task to #9008 to add a make tool target that builds the release-notes tool if not present and runs it with the expected args. [We should also update this guidance in the release docs](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#polish-release-notes).

File: hack/tools/release/notes.go
Comment (sbueringer) [reply hack/tools/release/notes.go]: thx, fixed

**Review by killianmuldoon — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**
